### ISSUE TITLE
"Duplicate this node" feature

### ISSUE THREAD

**Issue by justparking** [museumsvictoria/nodel#203]
Being able to duplicate a node (basically entire contents of node folder) but with a different name would be really useful.

Being able to do some simple substitutions or editing of the new `nodeConfig.json` at the same time would be nice too.

**Comment by paxofsnax** [comment]
Yes would be very helpful. I do this manually from explorer... alot. be v.good to do this within Nodel.
